pkg/util/template/func.go

```diff
@@ -22,3 +23,47 @@ func MakeMap(pairs ...interface{}) map[string]interface{} {
 
 	return out
 }
+
+// Add returns the sum of a and b.
```

Comment:
This seems copyed from https://github.com/hashicorp/consul-template/blob/de2ebf4/template_functions.go#L727-L901.

Let's keep it simple and include function libraries instead (e.g. [sprig](https://masterminds.github.io/sprig/)).